services/localfile/server/localfile_test.go

```diff
@@ -1576,134 +1560,236 @@ func TestCopy(t *testing.T) {
 						Filename: "/tmp/foo/../../etc/passwd",
 					},
 				},
-				Bucket: fmt.Sprintf("file://%s", filepath.Dir(f1.Name())),
-				Key:    filepath.Base(f1.Name()),
+				Bucket: fmt.Sprintf("file:///some-folder"),
+				Key:    "some-source-file",
 			},
-			wantErr: true,
 		},
 		{
 			name: "Bad bucket",
 			req: &pb.CopyRequest{
 				Destination: &pb.FileWrite{
 					Attrs: &pb.FileAttributes{
-						Filename: filepath.Join(temp, "/file"),
+						Filename: fmt.Sprintf("file:///some-folder"),
 					},
 				},
 				Bucket: "fil://not-a-path",
-				Key:    filepath.Base(f1.Name()),
+				Key:    "some-source-file",
 			},
-			wantErr: true,
 		},
 		{
 			name: "Bad key",
 			req: &pb.CopyRequest{
 				Destination: &pb.FileWrite{
 					Attrs: &pb.FileAttributes{
-						Filename: filepath.Join(temp, "/file"),
+						Filename: fmt.Sprintf("file:///some-folder"),
 					},
 				},
 				Bucket: "file://not-a-path",
-				Key:    filepath.Base(f1.Name()),
+				Key:    "some-source-file",
 			},
-			wantErr: true,
 		},
-		{
-			// Same here. We don't need to test all finalization permutations as TestWrite did that.
-			name: "Fail due to overwrite = false",
```

Comment:
extract it to separate test, because it was too hard to keep so different cases in the same test